src/platform/auth.ts

```diff
@@ -344,10 +350,10 @@ export const findUser = (
 						},
 					],
 				},
-				$select: ['id', 'actor', 'username', 'password'],
+				$select,
 			},
 		})
-		.then(([user]: AnyObject[]) => user);
+		.then(([user]: [T] | []) => user);
```

Comment:
```suggestion
		.then(([user]: [T?]) => user);
```

Reply:
I was hesitant about using this, since that's not 100% the same, but on second look I like that it's DRYer. Will change.